src/main.rs

```diff
@@ -1,12 +1,21 @@
+use crate::service::api::routes;
+use actix_web::{App, HttpServer};
+use constants::server_constants;
 use env_logger::Env;
 use log::info;
+
 pub mod application;
+pub mod constants;
 pub mod service;
 
-#[tokio::main]
-async fn main() {
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
     env_logger::Builder::from_env(Env::default().default_filter_or("debug")).init();
     info!("Launching app!");
-    // application::run();
     application::start().await;
+    info!("Starting web api");
+    HttpServer::new(|| App::new().service(routes::hello::hello))
```

Comment:
if we are keeping the application level, this should be there

Reply:
Done!